src/app/components/code/index.html

```diff
@@ -0,0 +1,39 @@
+<sky-demo-page
+  pageTitle="Code"
+  navTitle="Code"
+  skyuxModule="SkyCodeModule"
+  packageName="@blackbaud/skyux-lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code component formats text as a code snippet within larger blocks of text. Within the block of text, wrap <sky-code>sky-code</sky-code> tags around the code snippet. To format longer code snippets that stand alone as separate blocks of text, use the <a routerLink="/components/code-block">code block component</a> instead.
+    </p>
+    <p>
+      If a code snippet has text within <sky-code>&lt;</sky-code> and <sky-code>&gt;</sky-code> characters, then the component automatically appends a closing tag unless the code snippet is valid HTML. To prevent this, use "&#38;lt;" to escape <sky-code>&lt;</sky-code> and "&#38;gt;" to escape <sky-code>&gt;</sky-code>. For example, escape the <sky-code>&lt;</sky-code> and <sky-code>&gt;</sky-code> characters in the code snippet <sky-code>public Task&lt;Generic> MethodName()</sky-code> or the component automatically appends <sky-code>&lt;/Generic></sky-code> to the end: <sky-code>public Task&lt;Generic> MethodName()&lt;/Generic></sky-code>.
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the code component, you must set up the code block plugin. To install the plugin, run <sky-code>npm install @blackbaud/skyux-builder-plugin-code-block --save --save-exact</sky-code>. Then add the plugin to your <sky-code>skyuxconfig.json</sky-code> file: <sky-code>"plugins": ["@blackbaud/skyux-builder-plugin-code-block"]</sky-code>.
+      </sky-alert>
+    </p>
+  </sky-demo-page-summary>
+
```

Comment:
Our installation section is missing the steps for adding the module to app-extras.module.ts and adding the plugin to skyuxconfig.json as described in the Stache docs. Looks like we have that same problem with the code block docs. It seems like we're leaving out 2 pretty big steps for both components because the Installation section is cobbled together outside of the topic's index.html file. @Blackbaud-BobbyEarl, what do you think about these missing steps? Should we add them to overview section to make sure that they are at least on the page?.

Reply:
Yeah, we should add them.

Reply:
I don't think we can get those steps to appear in the Installation section, so I think the best we can do is include them in the page's opening section. But let me know if you think we should do something different.

Reply:
I should've hit all the points on this with the latest commit, and some gaps are filled by the existing note. John or Bobby, let me know if I missed something. Please and thanks!